websites/D/dod Games/presence.ts

```diff
@@ -0,0 +1,254 @@
+const presence = new Presence({
+	clientId: "937393073539911730",
+});
+
+let isInGame = false,
+	timeStarted = Date.now();
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "logo",
+		},
+		playerName = document.querySelector(
+			"#component_top_right_loginasname_text"
+		).textContent,
+		urls = document.querySelectorAll("#topBarDownLink");
+	let openRooms = 0;
+
+	for (const url of urls) {
+		if (url.textContent.includes("&room=")) openRooms++;
+
+		// Draw It
+		if (url.textContent.includes("game=drawit&room=")) {
+			if (!isInGame) {
+				timeStarted = Date.now();
+				isInGame = true;
+			}
+
+			let round = document.querySelector("#round b").textContent;
+			round = round.replaceAll(" ", "");
+			const scores = document.querySelectorAll("#playersContainer .player");
+			for (let i = 0; i < scores.length; i++) {
+				if (scores[i].querySelector(".name").textContent === playerName) {
+					if (
+						document.querySelector<HTMLDivElement>("#drawtools").style
+							.display !== "none"
+					) {
+						presenceData.smallImageKey = "molivaki";
+						presenceData.smallImageText = "Ζωγραφίζει";
+					}
+
+					presenceData.state = `Σκορ: ${
+						scores[i].querySelector(".score").textContent
+					} | Γύρος: ${round} | Θέση #${
+						parseInt(scores[i].getAttribute("index")) + 1
+					}/${scores.length}`;
+				}
+			}
+			if (scores.length < 10) {
+				presenceData.buttons = [
+					{
+						label: "Join Game",
+						url: url.textContent,
+					},
+				];
+			}
+			presenceData.details = "Παίζει Ζωγράφισέ Το";
+			presenceData.startTimestamp = timeStarted;
+			break;
+		} else if (url.textContent.includes("game=quiz&room=")) {
+			// Quiz
+			if (!isInGame) {
+				timeStarted = Date.now();
+				isInGame = true;
+			}
+			presenceData.buttons = [
+				{
+					label: "Join Game",
+					url: url.textContent,
+				},
+			];
+
+			const scores = document.querySelectorAll(
+				"#playersContainer .numberAndPlayerContainer"
+			);
+			for (let i = 0; i < scores.length; i++) {
+				if (
+					scores[i].querySelector(".quiz_playerName").textContent === playerName
+				) {
+					presenceData.state = `Σκορ: ${
+						scores[i].querySelector(".quiz_playerPoints").textContent
+					} | Γύρος: ${
+						parseInt(document.querySelector("#countRound").textContent) || "–"
+					} | Θέση #${parseInt(
+						scores[i].querySelector(".quiz_playerNumber").textContent
+					)}/${scores.length}`;
+				}
+			}
+			presenceData.details = "Παίζει Κουίζ";
```

Comment:
Same applies here. If the site says "Quiz", it should be "Παίζει Quiz", not entirely in Greek. The only time it would be otherwise which is UNLIKELY is if you are localizing it into other languages.

Reply:
Same here, it's called "Κουίζ" in dod.gr. Also there is no english version of the site, so it's highly unlikely the presence will get localized.